packages/story-editor/src/components/audioPlayer/index.js

```diff
@@ -86,19 +91,43 @@ function AudioPlayer({ title, src, mimeType, tracks = [], audioId }) {
 
     if (isPlaying) {
       player.pause();
-      setIsPlaying(false);
     } else {
-      player
-        .play()
-        .then(() => setIsPlaying(true))
-        .catch(() => {});
+      player.play();
```

Comment:
`.play()` returns a Promise, which means it can get rejected. Which is why we'd need the `catch(() => {})` here to avoid throwing errors.

See https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play